src/hooks/useTxDetails/useTxDetails.ts

```diff
@@ -58,50 +56,27 @@ export const getTransferByAsset = (tx: Tx, asset: Asset) =>
   tx.transfers.find(t => t.caip19 === asset.caip19)
 
 export const isSupportedContract = (tx: Tx) =>
-  tx.data?.method ? SUPPORTED_CONTRACT_METHODS.has(tx.data?.method) : false
+  Object.values(ContractMethods).includes(tx.data?.method as ContractMethods)
```

Comment:
We should still check for `undefined` here, else we might cast `undefined` as `ContractMethods`.

Reply:
Added a test for this. Casting `undefined` as `ContractMethod` won't add it to the type and resulting list.